x-pack/test_serverless/functional/test_suites/common/discover/group1/_discover_histogram.ts

```diff
@@ -67,7 +67,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await PageObjects.common.navigateToApp('discover');
       await PageObjects.discover.waitUntilSearchingHasFinished();
       // this is the number of renderings of the histogram needed when new data is fetched
-      let renderingCountInc = 1;
+      let renderingCountInc = 3;
```

Comment:
This behavior (which is not great I agree) didnt cause any rerenders. What have we changed which causes these rerenders now? 🤔

Reply:
in charts, we removed the debounce effect on resize. This mean that every single change in size is propagated and causes the chart to resize/rerender. Actually, nothing bad because the resize is something that doesn't happen so much in our apps so this doesn't cause any major performance problem.
In the functional test discover starts with the 2 ugly rows for the filter/query bar, but then changes to 1 single row when the focus goes to the filter bar. This triggers 2 resize events with different sizes in the chart (not sure why two but possibly due to some delay/effects in the various components on the filter bar.